### PR TITLE
Ensure the event examples and matrix.org assets are tested

### PR DESCRIPTION
**Pull request by turt2live**
Otherwise we go nearly a week without realizing the build is failing due to bad schemas/examples. This also helps us ensure a PR is up to par.

### CODE REVIEW

File: event-schemas/schema/m.room.encrypted [hunk 1]
```diff
@@ -14,7 +14,9 @@ properties:
         type: string
         enum:
```
Comment (ara4n) [inline event-schemas/schema/m.room.encrypted]: i'm confused - there should only be one valid value for each of olm & megolm?  according to js-sdk that's `m.olm.v1.curve25519-aes-sha2` and `m.megolm.v1.aes-sha2`.  Why keep the broken ones here at all?

Comment (turt2live) [reply event-schemas/schema/m.room.encrypted]: because I have no idea how encryption works - have removed them.

**Comment by turt2live** [comment]
lgtm'd out of band by @ara4n

**Comment by ara4n** [comment]
i know absolutely nothing about Circle, but the patch looks plausible - for infrastructural stuff like this I would forge ahead without getting too blocked on review.

**Pull request merged**